src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -59,17 +62,24 @@ internal class BindingProcessImpl : IBindingProcess
             IBindingConfigProvider bindingConfigProvider,
             SonarLintMode bindingMode,
             bool isFirstBinding = false,
-            IThreadHandling threadHandling = null /* overrideable for testing */)
+            IThreadHandling threadHandling = null /* overrideable for testing */,
+            IFileSystem fileSystem = null,
+            IExclusionSettingsStorage exclusionSettingsStorage = null
```

Comment:
If we are always receiving it from the caller, it shouldn't be null. The params above are nullable because they have a default value passed in the other CTOR